index.Rmd

```diff
@@ -89,107 +93,101 @@ Scroll down the license with enter. Accept License (i.e. type yes). Use default
 
 Scroll down the license with enter. Accept License (i.e. type yes). Use default install location (i.e. press Enter). When prompted to prepend the Conda install location to path agree (i.e. type yes). This makes the Python interpreter of Conda the standard Python interpreter and allows you to run Conda commands from command line. -->
 
-This will install Miniconda into `~/miniconda3`. Finally, restart your terminal to be able to use `conda` in the terminal. Next, let's see how to use Conda in case you want make new virtual environments by yourself, or install packages after creating the environment.
+This will install Mamba into `~/mamba`. Finally, restart your terminal to be able to use `mamba` and `conda` in the terminal. Next, let's see how to use Mamba in case you want make new virtual environments by yourself, or install packages after creating the environment.
 
-### Conda usage
-Conda creates isolated environments with sets of packages, that do not interfere with your base Python or with other conda environments. To create an environment:
+## Mamba usage
+Mamba creates isolated `conda` environments with sets of packages, that do not interfere with your base Python or with other `conda` environments. To create an environment:
```

Comment:
Same goes here, `conda` is a command, but in this case we are not talking about the command, but about the *Conda* environments.